File: src/main/java/br/com/unopay/api/util/StringJoiner.java
package br.com.unopay.api.util;

import java.util.Collection;
import java.util.stream.Collectors;

public class StringJoiner {
    private static final String DEFAULT_DELIMITER = ",";

    public static String join(Collection<String> strings) {
        return join(strings, DEFAULT_DELIMITER);
    }

    public static String join(Collection<String> strings, String delimiter) {
        if(strings != null && !strings.isEmpty()) {
            return strings.stream().collect(Collectors.joining(delimiter));
        } else {
            return "";
        }
    }
}

File: src/main/scala/br/com/unopay/api/util/StringJoiner.scala
package br.com.unopay.api.util

import java.util

import scala.collection.JavaConverters._

object StringJoinerS {
    val DEFAULT_DELIMITER : String = ","

    def join(strings: util.Collection[String] ): String = {
        join(strings, DEFAULT_DELIMITER)
    }
     def join(strings: util.Collection[String], delimiter: String) : String = {
        if (strings != null && !strings.isEmpty) {
            strings.asScala.mkString(delimiter)
        } else {
            ""
        }
    }
}